frontend/src/components/CountryLink.tsx

```diff
@@ -15,8 +16,10 @@ export const CountryLink: FC<TCountryLink> = ({ countryCode, className = "", chi
   const slug = getCountrySlug(countryCode, countries);
   if (!slug) return <>{children}</>;
   return (
-    <Link href={`/geographies/${slug}`}>
-      <a className={`flex items-center underline ${className}`}>{children}</a>
+    <Link
+      href={`/geographies/${slug}`}
+      className={`flex items-center underline ${className}`}>
```

Comment:
I am getting an error because the property 'className' does not exist on the typings for 'Link'